Command/CleanTransactionCommand.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace IDCI\Bundle\PaymentBundle\Command;
+
+use IDCI\Bundle\PaymentBundle\Entity\Transaction;
+use IDCI\Bundle\PaymentBundle\Payment\PaymentStatus;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+class CleanTransactionCommand extends ContainerAwareCommand
+{
+    private const DEFAULT_HOUR_DELAY = 24;
+
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setName('app:transaction:clean')
+            ->setDescription('Clean old transaction that stay in pending or created status after a given delay (in hours)')
+            ->setHelp('Clean transaction that stay in pending or created status after a given delay (in hours)')
```

Comment:
Change use P1D for exemple

Reply:
Fixed in 7fa6997f3ebe3e6a72ec9dd9350cb28b780da9a5